scripts/updateUsers.ts

```
@@ -212,16 +61,162 @@ async function main() {
-            console.error(`Failed to update user ${user.address}:`, error);
+
+      // 1) Fetch remote data for the entire batch concurrently (limited by batch size)
+      const fetchPromises = batch.map(async (user) => {
+        const address = user.address;
+
+        // Perform a last minute check to prevent directory traversal vulnerabilities
+        if (/[^a-zA-Z0-9]/.test(address)) {
+          return { address, error: new Error('updateUsers(): address contains invalid characters') };
+        }
+
+        const apiUrl = (process.env.API_URL || 'https://solo.ckpool.org') + `/users/${address}`;
```

ESLint

[error] Replace `·(process.env.API_URL·||·'https://solo.ckpool.org')·+` with `⏎··········(process.env.API_URL·||·'https://solo.ckpool.org')·+⏎·········`

(prettier/prettier)